src/App.vue

```diff
@@ -50,9 +54,9 @@ setInterval(nextWidget, 10000);
         <WidgetPicto />
       </BaseWidget>
     </div>
-    <BaseWidget :class="$style.settings" v-if="isSettingsOpen">
-      <WindowSettings />
-    </BaseWidget>
+    <WindowEmergency :class="$style.emergency" v-show="isEmergencyWindowOpen" />
```

Comment:
```suggestion
    <WindowEmergency :class="$style.emergency" v-if="isEmergencyWindowOpen" />
```
v-showは頻繁に表示/非表示が切り替わる場合に使う